docs/stories/13 - Cold Hands, Stone Heart.adoc

```diff
@@ -0,0 +1,135 @@
+
+Cold. Very cold. The first thing I feel isn't the pain in my ribs or the throbbing in my head, but the chill that surrounds me. So much more than before. The air is dry and thin and sharp, but still. I expect pain. Why? I look down at my hands, pressed into the ground. No, that's wrong. They're pressed into the expanse of white lying atop the ground, my arms pressed through to the coolness beneath. My fingers are already growing numb, but I can feel the soil between them, hidden by this blanket of...
+
+_...snow._
+
+The word comes to me. I let the word wash over me like the chill, mountain air.
+
+These are mountains. I hadn't known mountains before. Fascinating. I have much reason to question myself, question my memories, or lack of them, but I have little reason to doubt this. I turn my head to examine my surroundings. These large evergreen oak trees, oogashi? The name comes to me. They surround this clearing. Overhead the stars shine through a clear, velvet sea on which sails an enormous silver moon-
+
+-I'm in the trees before I realize it, my breathing ragged. I hide from the moon's sight, but I don't know why. I don't question it. Am I afraid? I need to focus!
+
+My breathing quiets. There is no reason to fear here. Ahead of me a soft color, pink, so pale as to be almost white. A flower. It shouldn't be here now; it seems out of place. Like I am. It must have bloomed before the snow came. It never fell on its own, sheltered somehow from the mountain winds, until the frost and ice enclosed it. I see five petals. At their center a collection of strands reach out, no longer pale but vibrant and bright, the image of vitality were it not for the ice that encased them. Perhaps that is why the flower is preserved? Did the cold come on so quickly the blossom had no chance to fall?
+
+The mystery calms me before I realize it.
+
+The ice traces each line of each petal, the stamen held in an eternal dance above them. I reach out towards the mume blossom. I need it. My hand closes around it.
+
+It shatters in my grasp.
+
+There is a gasp behind me. Remembering the moon, I whirl around. I can't have been found so quickly!
+
+But there is nothing to fear. Only a girl. Young. Small. How old am I, myself? I don't know.
+
+"Who are you?" she asks.
+
+I have no answer so I say nothing.
+
+She seems unafraid and comes close, all the while asking questions. "Where did you come from?"
+
+I don't answer. I'm not sure I know. Above?
+
+"How did you get here?"
+
+I feel like I fell, but don't know how to explain. I stay silent.
+
+"What happened to your clothes?"
+
+I look down at my nakedness. It is more inconvenient than embarrassing.
+
+"Aren't you cold?"
+
+"A little."
+
+There is no surprise when I finally speak. She just shrugs. "I'm Rin. Come with me."
+
+Dropping the shattered blossom, I follow her.
+
+'''
+
```

Comment:
Please add a `{empty} +` after the horizontal rule, with a blank line before and after it, otherwise the ruler is directly attached to the next line in the PDF.